.github/PULL_REQUEST_TEMPLATE_PROPOSED-UPDATES.md

```diff
@@ -0,0 +1,130 @@
+      - [ ] SDK of .NET (need service team to ensure code readiness)
+      - [ ] SDK of Python
+      - [ ] SDK of Java
+      - [ ] SDK of Js
+      - [ ] SDK of Go
+      - [ ] PowerShell
+      - [ ] CLI
+      - [ ] Terraform
+      - [ ] No refresh required for updates in this PR
+
+### Contribution checklist:
+- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
+- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
+- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)
+
+If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).
+
+### ARM API Review Checklist 
+
+> **Applicability**: :warning: 
+>
+> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
+> - Change to data plane APIs 
+> - Adding new properties 
+> - All removals
+
+Otherwise your PR may be subject to ARM review requirements. Complete the following:
+- [ ] Check this box if any of the following apply to the PR so that label “WaitForARMFeedback” will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
+  - Adding a new service
+  - Adding new API(s)
+  - Adding a new API version
+    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits.
+
+- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.
+
+- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 
+
+### Breaking Change Review Checklist 
+If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
+- [ ] Removing API(s) in a stable version
+- [ ] Removing properties in a stable version
+- [ ] Removing API version(s) in a stable version
+- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
+- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))
```

Comment:
Maybe we don't need these bullets at all and just refer them to document (which you are already doing).

Reply:
@JeffreyRichter shouldn't we give a quick list of what qualifies as a breaking change so teams know to check  [Breaking Change Policy](http://aka.ms/bcforapi)?

Reply:
No, the will have a good idea already.